gems/pending/util/miq-hash_struct.rb

```diff
@@ -55,7 +55,8 @@ def method_missing(m, *args)
 
   private
 
-  def respond_to_missing?(*)
-    true
+  def respond_to_missing?(*args)
```

Comment:
`(sym, *)` would be neater; saves the `.first` down below (and an array allocation, for that matter)